Tools/build/generate_sbom.py

```diff
@@ -82,6 +82,14 @@ def spdx_id(value: str) -> str:
     return re.sub(r"[^a-zA-Z0-9.\-]+", "-", value)
 
 
+def error_if(value: bool, error_message: str) -> None:
+    """Prints an error if a comparison fails along with a link to the Dev Guide"""
```

Comment:
Normally just "devguide" for short:
```suggestion
    """Prints an error if a comparison fails along with a link to the devguide"""
```